scheduler/bin/help-delete-temporary-clusters

```diff
@@ -1,19 +1,23 @@
 #!/usr/bin/env bash
 
-# Usage: ./bin/help-delete-temporary-clusters <zone>
+# Usage: ./bin/help-delete-temporary-clusters <project> <zone>
 #   Delete all temporary clusters within a zone. The sibling scripts here mark clusters they create.
 #   This is intended to be used by other scripts and not directly.
+#   <project> is a gcloud project.
 #   <zone> can be a zone. E.g., us-central1-a
 
 
 set -e
 
-ZONE=$1
+PROJECT=$1
+ZONE=$2
+
+gcloud="gcloud --project $PROJECT"
 
 # Nuke all existing temporary clusters; don't want to keep on making more idle clusters each time you invoke this.
 echo "---- Deleting any existing temporary clusters."
-gcloud container clusters list --filter 'resourceLabels.longevity=temporary'
-for i in `gcloud container clusters list --filter 'resourceLabels.longevity=temporary' --format="value(name)"`
+$gcloud container clusters list --filter 'resourceLabels.longevity=temporary'
```

Comment:
If we're setting this up with automatic names and expecting a project to be shared by multiple users, shouldn't this be changed to
#1: Label the cluster with the user in question.
#2: Only delete clusters by the same user
#3: Consider prompting a user if they want to delete the clusters before deleting them. (with a -f option for automated purposes)
#4: Delete in parallel. Its pretty slow.

Reply:
I agree, those all sound like good ideas.

My goal for this PR was just to get the scripts working, without any functionality improvements. As is, they're not respecting the project being passed in at the top level, so many of the gcloud commands fail. This PR gets them to where they run successfully using the project provided.